string my_command

---

string my_response
